### PR TITLE
1.6.2 UI improvement

### PR DESCRIPTION
**Pull request by fedorenksei**
## Task

[Card in GitHub project](https://github.com/users/fedorenksei/projects/2/views/1?pane=issue&itemId=57085763)

## What type of PR is this? (check all applicable)

- [ ] 🍕 Feature
- [x] 🐞 Bug Fix
- [x] 🎨 Style
- [x] 👨‍💻 Code Refactor
- [ ] ✅ Test
- [ ] 📝 Documentation Update
- [ ] 🛠 Config
- [ ] ✨ Optimization
- [ ] 🤷‍♂️ Other: ...

## Description

- Removed duplicated buttons: "Log out" in Welcome, "To welcome" on Main
- Used outlet and took out `<Footer />` and `<Header />` from the page components
- Corrected some texts
- Made the footer attached to the page bottom
- UPD: Remove page reloading when clicking in buttons

## Todo

- Enhance Not found page
- Refactor architecture: move some components of them from pages to widgets or features

## Added tests?

- [x] ✅ Yes, tests are adjusted to the changes
- [ ] 🙅‍♂️ No, they aren't needed
- [ ] 🙋‍♂️ No, because I need help

## Added to documentation?

- [ ] 📝 Yes, in README.md
- [x] 🙅‍♂️ No documentation needed

### CODE REVIEW

File: src/app/model/router.tsx [hunk 1]
```diff
@@ -1,33 +1,49 @@
...
+    path: '',
     element: (
-      <>
-        <Welcome />
-        <PopupHandler />
-      </>
+      <div className="flex flex-col h-full">
+        <Header />
+        <div className="flex-grow">
+          <Outlet />
```
Comment (color-zebra) [inline src/app/model/router.tsx:17]: Nice decision to use outlet here

File: src/pages/Header/index.tsx
Comment (color-zebra) [inline src/pages/Header/index.tsx:1]: I think header should be placed in widgets folder

File: src/pages/Main/Main.tsx [hunk 2]
```diff
@@ -1,36 +1,21 @@
...
-          <Link to="/">
-            <Button>{words.toWelcome}</Button>
-          </Link>
-        </div>
-        <GraphQL />
-      </div>
-      <Footer />
-    </>
+    <div className="p-12 mt-20">
+      <GraphQL />
```
Comment (color-zebra) [inline src/pages/Main/Main.tsx:18]: Great idea, this will make the pages layer thinner

Comment (Color-zebra) [inline src/pages/Main/Main.tsx:18]: Great idea, this will make the pages layer thinner

File: src/pages/Welcome/Welcome.tsx [hunk 3]
```diff
@@ -30,35 +25,22 @@ export const Welcome = () => {
             >
               <Button>{words.signin}</Button>
```
Comment (color-zebra) [inline src/pages/Welcome/Welcome.tsx]: Nesting button and link is not valid, we should style link as button or use button with onClick(() => {...some naviagtion...}) instead Link

File: src/pages/Main/Main.tsx (hunk 2 above)
Comment (Color-zebra) [inline src/pages/Main/Main.tsx:18]: Great idea, this will make the pages layer thinner

**Review by color-zebra — APPROVED** [review]
I am very glad that our project is alive! :)

**Review by Nickbis — APPROVED** [review]
Nice to see this project. I think all the changes improve it. Everything is fine

**Pull request merged**